docs/releases/v2020.2.rst

```diff
@@ -0,0 +1,190 @@
+has to be distributed to all participating nodes using the site.conf.
+
+Responsive status page
+======================
+
+The status page design is now responsive and reflows better on mobile devices.
+
+Primary domain code
+===================
+
+The primary domain code is now visible on the node status page as well as in the respondd information
+emitted by the node.
+
+Logging
+=======
+
+- The new *gluon-logging* package allows to configure a remote syslog server using the site.conf.
+  This package can only be included when *gluon-web-logging* is excluded.
+
+Peer cleanup in fastd
+=====================
+
+fastd peers and groups are now removed on update in case they do not exist in the new site configuration.
+To preserve a custom peer across updates, add the *preserve* key to the peers UCI configuration and set it to ``1``.
+
+
+Bugfixes
+--------
+
+- The WAN MAC address now matches the one defined in OpenWrt if VXLAN is enabled for the selected domain.
+
+- *gluon-reload* now reloads all relevant services.
+
+- Disable outdoor mode and enabling meshing in the config mode can now be performed in a single step.
+
+- Fixed section visiblity with enabled outdoor mode in config mode.
+
+
+Site changes
+------------
+
+site.mk
+=======
+
+* Starting with version 19.07 OpenWrt ships the urngd entropy daemon by default.
+  It replaces the haveged daemon, for which we removed the support in Gluon. Remove ``haveged`` from your package selection.
+
+
+Internal
+--------
+
+Editorconfig
+============
+
+Gluon now ships a *editorconfig* file to allow compatible editors to automatically apply key aspects of Gluons code style.
+
+Continuous Integration
+======================
+
+* Jenkins
+
+  - The CI now has a test stage to verify Gluons runtime functionality.
+
+* GitHub Actions
+
+  - GitHub actions is now enabled for the Gluon project, build-testing all available targets.
+
+Build system
+============
+
+  - Source code minification can now be skipped by enabling the GLUON_MINIFY flag.
+
+  - Enabling the GLUON_AUTOREMOVE flag will remove package build directories after they are built.
+    This reduces space consumption at the expense of subsequent builds being slower.
+
+
+Known issues
+------------
+
+* Out of memory situations with high client count on ath9k.
+  (`#1768 <https://github.com/freifunk-gluon/gluon/issues/1768>`_)
+
+* The integration of the BATMAN_V routing algorithm is incomplete.
+
+   - | Mesh neighbors don't appear on the status page. (`#1726 <https://github.com/freifunk-gluon/gluon/issues/1726>`_)
+     | Many tools have the BATMAN_IV metric hardcoded, these need to be updated to account for the new throughput
+     | metric.
+
+   - | Throughput values are not correctly acquired for different interface types.
+     | (`#1728 <https://github.com/freifunk-gluon/gluon/issues/1728>`_)
+     | This affects virtual interface types like bridges and VXLAN.
```

Comment:
```suggestion
  - Mesh neighbors don't appear on the status page. (`#1726 <https://github.com/freifunk-gluon/gluon/issues/1726>`_)
    Many tools have the BATMAN_IV metric hardcoded, these need to be updated to account for the new throughput
    metric.

  - Throughput values are not correctly acquired for different interface types.
    (`#1728 <https://github.com/freifunk-gluon/gluon/issues/1728>`_)
    This affects virtual interface types like bridges and VXLAN.
```
Putting the sub-points into a blockquote may be a nice hack to fix the broken margins in the RTD theme, but I think it would be better if we could just fix the CSS (which I hope is possible on readthedocs.io, and is also unrelated the release notes)